modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/linear-segregated/steady-transient-compare/compare.cmp

```diff
@@ -0,0 +1,3 @@
+TIME STEPS relative 1.0 floor 0.0
+
+ELEMENT VARIABLES relative 5e-5 floor 1e-6
```

Comment:
note that you dont need a .cmp file if you are not assigning tolerances on a per-variable basis. you can set rel_err / abs_zero in the tests config

Reply:
I did this just for the time step tolerance.

Reply:
oh nice